ejb/WFLY-12190-http-server-to-server-config.adoc

```diff
@@ -0,0 +1,153 @@
+= EJB over HTTP server-to-server invocation
+:email:             tomasz.adamski@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+=== Issue
+* https://issues.jboss.org/browse/WFLY-12190
+
+=== Dev Contacts
+
+* mailto:tadamski@redhat.com
+
+=== QE Contacts
+
+=== Testing By
+[ ] Engineering
+
+[x] QE
+
+=== Affected Projects or Components
+EJBs
+
+== Introduction
+
+PRODMGT-268 (https://issues.redhat.com/browse/PRODMGT-268) added the ability to configure EJB invocations over HTTP protocol. Customers would like to use this feature in server-to-server configurations.
```

Comment:
As this is a community project can we please try and cross reference community issues?

Reply:
fixed